### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by a-laarabi**
# What?
In this repository, I developed the UI of the calculator.

### Highlights:
- I did make sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
- I used correct [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md).
- I documented my work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).
- I followed the list of [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md).
- I did import the calculator and use it in the main component (App.js).
- I test it in the browser; it render without problems.

Cheer and happy reviewing 🥇

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
-    <div className="App">
```
Comment (Deepakdanger) [inline src/App.js:6]: - [ ] The `App.js` component should be class-based.

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      <div className="button">7</div>
+      <div className="button">8</div>
+      <div className="button">9</div>
+      <div className="button operator">x</div>
+      <div className="button">4</div>
+      <div className="button">5</div>
+      <div className="button">6</div>
+      <div className="button operator">-</div>
+      <div className="button">1</div>
+      <div className="button">2</div>
```
Comment (Deepakdanger) [inline src/components/Calculator.js:20]: - [ ] The `Calculator.js` component should be class-based

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @a-laarabi, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DammyShittu — APPROVED** [review]
Hi @a-laarabi,

## APPROVED 🟢

## Highlights
✔️ Class components

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**